test/integration_testproj.go

```diff
@@ -23,41 +29,53 @@ type IntegrationTestProject struct {
 	t          *testing.T
 	h          *Helper
 	preImports []string
+	tempdir    string
+	env        []string
+	origWd     string
+	stdout     bytes.Buffer
+	stderr     bytes.Buffer
 }
 
-func NewTestProject(t *testing.T, initPath string) *IntegrationTestProject {
+func NewTestProject(t *testing.T, initPath, wd string) *IntegrationTestProject {
 	new := &IntegrationTestProject{
-		t: t,
-		h: NewHelper(t),
+		t:      t,
+		origWd: wd,
+		env:    os.Environ(),
 	}
-	new.TempDir(ProjectRoot)
+	new.makeRootTempDir()
 	new.TempDir(ProjectRoot, "vendor")
 	new.CopyTree(initPath)
-	new.h.Setenv("GOPATH", new.h.Path("."))
-	new.h.Cd(new.Path(ProjectRoot))
+	if runtime.GOOS == "darwin" {
+		new.Setenv("GOPATH", filepath.Join("/private", new.tempdir))
```

Comment:
OHHHH THE STUPID `/private` LEADER

Yeah, that's one of those things that bites me, I curse at my mac for a while, then I promptly forget about. Sorry - that must've been a giant pain to track down through runs against a slow CI system 😦 

I think we may get a bit more portability if you first check that the first element of `new.tempdir` is `/tmp`. Other than that, though, this strikes me as a sane way of compensating for this problem, at least for now.